cmd/traefik/traefik.go

```diff
@@ -173,7 +174,8 @@ func runCmd(globalConfiguration *configuration.GlobalConfiguration, configFile s
 		acme.Get().SetConfigListenerChan(make(chan types.Configuration))
 		svr.AddListener(acme.Get().ListenConfiguration)
 	}
-	svr.Start()
+	ctx := server.ContextWithSignal(context.Background())
```

Comment:
should this be _just_ in the main (or in `cmd/traefik` in that case). The `server` package doesn't really have to care about signal handling.

Reply:
I'm ok with this, I will move ContextWithSignal